### ISSUE TITLE
r_lodcurveerror not working in Opengl1 (classic renderer)

### ISSUE THREAD

**Issue by AtifArshad1995** [ioquake/ioq3#383]
Hi Devs, i was trying to get the far away curved surfaces to look smooth. So i put r_lodcurveerror=10000 (max value) in autoconfig.cfg but it wasn't working. 

But when i changed the renderer to opengl2, it worked and far away surfaces did not loose their curves. Thus i concluded that it is a behaviour of classic render.

I am not sure this behaviour is present in original Quake3.exe but it is obvious that it is a bug so can u guys fix this issue.

**Comment by zturtleman** [comment]
`r_lodcurveerror` cvar is cheat protected. Are you loading maps using the `devmap q3dm1` console command?

**Comment by AtifArshad1995** [comment]
No i wasn't using devmap command. After using the command, it is working. However i didn't need to use devmap command to make it work in opengl2. 

So it means that r_lodcurveerror is cheat protected only in opengl1 but not in opengl2? Is it possible to exclude r_lodcurveerror from cheat protection as it is in opengl2

**Comment by zturtleman** [comment]
`r_lodcurveerror` is cheat protected in opengl2 renderer too. However it turns out the opengl2 renderer doesn't use `r_lodcurveerror` in most cases because it uses a faster rendering method (vertex buffer).

It seems that either `r_lodcurveerror` should not be cheat protected or OpenGL2 should go out of it's way to actually use `r_lodcurveerror`. Though it's unclear if increasing or decreasing it is a cheat. Maybe it should be non-cheat with a limited range (250, the default, to 10000).

**Comment by AtifArshad1995** [comment]
Yes i think it should be non-cheat because it doesn't even provide any gameplay advantage. If things like r_subdivisions and r_picmip (which provide significant advantage) are non-cheats then i see no reason why r_lodcurveerror should be cheat protected.

**Comment by Jon8RFC** [comment]
I just learned of this setting and wanted to crank it up to 10000 as well, but can't.

I'm no software developer, but is it as simple as me turning a flag off to in the code to allow this?
